spec/features/logged_in_user_can_log_out_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.feature 'Logged in user logs out' do
+  scenario 'they see the visitor options' do
+    user = create(:user)
+    # As a logged in user
+    visit '/'
```

Comment:
It would be cleaner to go straight to '/login', since we aren't testing login functionality in this test.